README.md

```diff
@@ -17,23 +17,22 @@ function MyButton () {
 }
 MyButton.prototype = Object.create(Nanocomponent.prototype)
 
-MyButton.prototype._render = function (color) {
-  this._color = color
+MyButton.prototype._render = function () {
```

Comment:
in general I'm not a fan of exposing `this` as the API. For `nanocomponent` I guess it makes a lot of sense b/c inheritance; but for micro I think it would be nice if people aren't aware we're doing anything with prototypes at all. Also b/c not using prototypes tends to be a bit friendlier to people that are new to JS - which Nanocomponent is not necessarily aimed at

But yeah that's not about this module I guess haha :D